src/clusterfuzz/_internal/crash_analysis/stack_parsing/stack_symbolizer.py

```diff
@@ -525,6 +528,61 @@ def _lkl_line_parser(self, line):
 
     return None, None, None, None, None
 
+  def _trusty_line_parser(self, line):
+    """Parses line for memory_space, offset, and addr"""
+    match = STACK_TRACE_LINE_TRUSTY.match(line)
+    if match:
+      memory_space, offset, addr = match.groups()
+      return memory_space, offset, addr
+
+    return None, None, None
+
+  def _extract_trusty_app_name(self, stacktrace):
+    #(app: keymaster)
+    match = re.compile(r'\(app:\s*(\w+)\)').search(stacktrace)
+    if match:
+      return match.group(1)
+
+    return ''
+
+  def _close_pipes(self):
+    for pipe in pipes:
+      pipe.stdin.close()
+      pipe.stdout.close()
+      pipe.kill()
+
+  def process_trusty_stacktrace(self, unsymbolized_crash_stacktrace):
```

Comment:
Are there any tests you can write for this?

Reply:
Yes, thanks for the suggestion! I've added a test with some sample stacktrace data now.

Reply:
Hi @oliverchang - Following up on our discussion, I removed the unit test for now (presubmits are OK without that unit test), with the intent to submit the test in a separate PR to further debug why it could be failing /gcbrun.